OmniKit/PumpManager/OmnipodPumpManager.swift

```diff
@@ -647,12 +647,14 @@ extension OmnipodPumpManager {
         
         #if targetEnvironment(simulator)
         let mockDelay = TimeInterval(seconds: 3)
+        let mockFaultDuringInsertCannula = false
```

Comment:
Please avoid "configuration in code". If it's truly something that needs to be user configurable, it should go into the UI. Otherwise, make the decision and implement it as simply as possible.

Reply:
This was simply a way to avoid having commented out optional developer test code which can drift out of date. This isn't something for users to configure and it doesn't make sense to make the decision to always fault when using the simulator.